src/main/java/school/lemon/changerequest/java/introduction/hw2/Task6.java

```diff
@@ -1,8 +1,19 @@
 package school.lemon.changerequest.java.introduction.hw2;
 
 
+import com.sun.xml.internal.fastinfoset.util.CharArray;
+
 public class Task6 {
     public static int[] reverse(int[] arr) {
-        return null;
+        if (arr == null) {
+            return null;
+        }
+        int reverseArr[] = new int[arr.length];
+        int n = 0;
+        for (int i = arr.length - 1; i >= 0; --i) {
+            reverseArr[i] = arr[n];
+            n++;
+        }
```

Comment:
You can create more complex `for` condition:
```java
for (int i = arr.length - 1, n=0; i >= 0 && n<arr.length; i--, n++) {
  reverseArr[i] = arr[n];
}
```

But, actually, this task could be solved with only one index variable.